scripts/run-hydra_cli-reference.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Exit script if you try to use an uninitialized variable.
+set -o nounset
+
+# Exit script if a statement returns a non-true return value.
+set -o errexit
+
+# Use the error status of the first failure, rather than that of the last item in a pipeline.
+set -o pipefail
```

Comment:
We use a [standard `set`](https://github.com/ory/hydra/blob/master/scripts/run-genswag.sh#L5) for this:

```suggestion
set -Eeuxo pipefail
```